client/src/components/_common/Button/Button.module.css

```diff
@@ -35,6 +35,11 @@
   composes: c-button--as-link from '../../../styles/components/c-button--new.css';
 }
 
+/* WP-50: Sets size of text to 1rem (16px) */
```

Comment:
```suggestion
```

Reply:
Yep! Not sure why I left this in.
I'll take it out.